reliability/stub-core/src/main/kotlin/com/datadog/android/core/stub/StubSDKCore.kt

```diff
@@ -91,6 +91,9 @@ class StubSDKCore(
     override val internalLogger: InternalLogger = StubInternalLogger()
 
     override fun registerFeature(feature: Feature) {
+        // Stop previous registered
+        featureScopes[feature.name]?.unwrap<Feature>()?.onStop()
```

Comment:
shouldn't we just stop core instead after each test? and stop the features in the `stop` method of the code.

Reply:
This part is only required because in one of the tests I override the previously started feature: in the `@BeforeEach` set up, we register a Logs feature, but in some tests I need to register a feature with a different configuration.